src/lf_mask.rs

```diff
@@ -128,15 +129,37 @@ fn decomp_tx(
         let lw = cmp::min(2, t_dim.lw);
         let lh = cmp::min(2, t_dim.lh);
 
```

Comment:
A `debug_assert!(t_dim.w == 1 << t_dim.lw && t_dim.w <= 16)` here makes it clear that `t_dim.w` is a power of 2 and `<= 16` and thus `CaseSet::<16, false>` results in an exhaustive `match` and thus always initializes the slice it is supposed to.

Note that `t_dim` always comes from a `static`, so this is known statically, which is why I think a `debug_assert!` rather than `assert!` suffices for safety.  Ideally we'd just do a full `assert!` and it'd get optimized out, but LLVM isn't smart enough to do that yet.